scripts/qa_crowbarsetup.sh

```diff
@@ -2221,6 +2221,12 @@ function enable_ssl_generic
             fi
             $p "$a['api']['protocol']" "'https'"
         ;;
+        magnum)
+            if ! iscloudver 7plus ; then
```

Comment:
since GM7 will not support magnum+https, we probably want a better check here.
e.g. test for existance of chef/data_bags/crowbar/migrate/magnum/101_add_ssl_attributes.rb